src/RestClient.ts

```diff
@@ -371,6 +371,12 @@ export class RestClient extends BaseRestClient {
     return this.postPrivate('spot/v4/cancel_orders', params);
   }
 
+  cancelAllSpotOrders(params: {
```

Comment:
if all properties in the object are optional, then params should be optional too
```suggestion
  cancelAllSpotOrders(params?: {
```

Reply:
Fixed, missed it. Thanks! :)